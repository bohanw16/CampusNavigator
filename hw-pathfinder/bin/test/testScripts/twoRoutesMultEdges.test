# find the shortest path with two routes

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3

AddEdge g n1 n2 5.555
AddEdge g n1 n3 1.111
AddEdge g n3 n2 2.222

FindPath g n1 n2