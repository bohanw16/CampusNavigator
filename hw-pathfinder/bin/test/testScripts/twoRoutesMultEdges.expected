# find the shortest path with two routes

created graph g

added node n1 to g
added node n2 to g
added node n3 to g

added edge 5.555 from n1 to n2 in g
added edge 1.111 from n1 to n3 in g
added edge 2.222 from n3 to n2 in g

path from n1 to n2:
n1 to n3 with weight 1.111
n3 to n2 with weight 2.222
total cost: 3.333